### PR TITLE
Configure yamllint

### PR DESCRIPTION
**Pull request by dato**
-----
Fixes the easy part of: #11

### CODE REVIEW

File: .yamllint.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    max-spaces-inside: 1  # Required by ansible-lint
+  octal-values:
+    forbid-implicit-octal: true  # Required by ansible-lint
+    forbid-explicit-octal: true  # Required by ansible-lint
+  quoted-strings:
+    quote-type: single
+    required: only-when-needed
+    extra-required:
+      - ^[0-9:]+$  # IPv6 addresses containing only numbers 0-9 are misparsed
+    extra-allowed:
```
Comment (dato) [inline .yamllint.yaml:22]: Hmm, @NightTsarina:

If I'm not reading wrong, with the current config, yamllint complains quotes are not needed in a line like this one:

```yaml
grafana__smtp__from_address: 'grafana@{{ host_fqdn }}'
```

Do you think we should allow quotes if `{{ .+ }}` is found in the string?

Comment (NightTsarina) [reply .yamllint.yaml:22]: It is correct quotes are not needed in that example (for YAML parsing). I am fine with either setting for yamllint

**Review by NightTsarina — APPROVED** [review]

File: .yamllint.yaml (hunk 1 above)
Comment (dato) [reply .yamllint.yaml:22]: Let's keep it simple then (no extra conf.), unless @mrfussion strongly objects.

**Review by mrfussion — APPROVED** [review]
LGTM. I agree with that

**Pull request merged**